defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-mysql_hardening_enabled: yes
+mysql_hardening_enabled: no
```

Comment:
I think disabling the role by default is counter-intuitive.
If I would use this role, I'd expect that it would work out-of-the-box (like os and ssh hardening), not that I should still have to activate it. Because that's already what I'm doing when I'm including this role into one of my playbooks.